### PR TITLE
Notification Metrics: Update descriptions, fix colours, remove extra stuff not needed

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Made a few adjustments:
- Added descriptions for all of the visualizations, and the board itself.
- Removed a bunch of queries that aren't needed
- Updated the colour scheme

### CODE REVIEW

File: grafana/mattermost-notification-metrics.json [hunk 1]
```diff
@@ -76,6 +76,7 @@
       }
     ]
   },
+  "description": "A dashboard displaying notification health metrics, and distributions around notifications that were not sent, both deliberate and indeliberate. \n\nThe dashboard is broken in half by Websocket events (POSTED events from the server to web clients) and Push events (events sent to the push proxy to be received by mobile clients).",
```
Comment (marianunez) [inline grafana/mattermost-notification-metrics.json]: nit: minor suggestion to change `broken` as at first read I misinterpreted as it was not working

```suggestion
  "description": "A dashboard displaying notification health metrics, and distributions around notifications that were not sent, both deliberate and indeliberate. \n\nThe dashboard is split in half by Websocket events (POSTED events from the server to web clients) and Push events (events sent to the push proxy to be received by mobile clients).",
```

File: grafana/mattermost-notification-metrics.json [hunk 2]
```diff
@@ -516,20 +481,32 @@
...
       "title": "Websocket Notification Stats",
-      "transformations": [],
       "type": "timeseries"
     },
     {
       "datasource": {
         "type": "prometheus",
         "uid": "${DS_PROMETHEUS}"
       },
+      "description": "Displays the total number of acknowledged notifiable events over the given range. If \"Bad range\" is shown, that means the counter was reset and the value cannot be accurately determined. It is advised to pick a date range in which the counter has not been reset.\n\nThe Sent number indicates the number of events that resulted in a notification to the end user, and not sent is the number of events where the system decided not to send the notification.\n\nIf shown, Unknown indicates the number of events where we could not determine the result of the event. This usually indicates an out-of-date Desktop App.",
```
Comment (marianunez) [inline grafana/mattermost-notification-metrics.json]: Should we explain when the counter is expected to reset?

File: grafana/mattermost-notification-metrics.json [hunk 3]
```diff
@@ -544,31 +521,44 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "Total Notification Errors"
+              "options": "Total Notifications Unknown"
```
Comment (marianunez) [inline grafana/mattermost-notification-metrics.json:524]: What does `Unknown` mean in this context? It's not always an error in this category? Which part is this in the dashboard?

**Review by marianunez — COMMENTED** [review]
@devinbinnie is this version up in the community/hub dashboards to see it live? It's hard to visualize where each section is from reading the json

File: grafana/mattermost-notification-metrics.json (hunk 3 above)
Comment (devinbinnie) [reply grafana/mattermost-notification-metrics.json:524]: The sent vs. not sent notifications for websocket:
image

**Comment by devinbinnie** [comment]
> @devinbinnie is this version up in the community/hub dashboards to see it live? It's hard to visualize where each section is from reading the json

Yes it is.

**Review by marianunez — APPROVED** [review]
Looks great, thanks @devinbinnie!

**Review by agnivade — APPROVED** [review]
Sorry I was on PTO.

**Pull request merged**